docs/release-notes/v11.6.0.md

```diff
@@ -0,0 +1,138 @@
+---
+id: v11.6.0
+title: Version v11.6.0 Release Notes
+sidebar_label: v11.6.0
+image: "img/release-note-link-preview.png"
+---
+
+_Sep 14th, 2023_
+
+## Console
+
+### Enhanced Project Workflow
+
+We are pleased to share with you a new BETA feature involving a significant architectural change on Mia-Platform Project structure.
+
+The **Enhanced Project Workflow** feature aims to provide a better developer experience when working in Console, enabling both [pipeline-based](/development_suite/deploy/pipeline-based/index.md) and [GitOps-based](/development_suite/deploy/gitops-based/index.md) deployment strategies.
+
+Visit the [Enhanced Project Workflow documentation](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) to discover all the benefits and new incoming features!
+
+:::info
+If you want to try this enhanced project workflow, please open a service request and ask for the support of a Mia-Platform referent.
+:::
+
+### Project Settings
+
+Inside the 'Overview' Area of a Project, you can now access a brand new section called 'Settings', where users with enough administrative permissions can view and manage the settings of their Project. This will speed-up and optimize the overall administration of Projects, which – until now – was only possible exclusively via CMS, making it no longer tedious or error-prone!
+
+![Project Settings](./img/overview/project-settings.png)
+
+For more information about Project Settings, [go to the official documentation](/console/project-configuration/project-settings.md).
+
+### Delete a Project
+
+To enable even more exhaustive management of the resources in Console, it is now possible to delete a project directly from its settings. 
+Simply go to the respective Advanced tab and the user can easily delete a project with all its resources. 
+
+For more information, [go to the official documentation](/console/project-configuration/delete-a-project.md).
+
+### Share Internal Company Marketplace items to other Companies with Marketplace Providers
+
+We are thrilled to announce that from now on a Company will be able to allow access to the resources of its Internal Company Marketplace to other Companies!  
+This can be done thanks to the introduction of a new Marketplace feature for Providers in Console. 
+
+You will be able to add a Marketplace Provider in your Company and manage the accessibility of Marketplace resources connected to that Provider directly from the Providers section in Console. All you need to do is to add a Marketplace Provider by clicking on the "Add Marketplace Provider connection" option – available in the Providers section – and then configure on the CMS all the Marketplace resources that you want to make available to other Companies. This can be done by specifying the providerId of the Marketplace Provider and editing the visibility property for each of those Marketplace resources.
```

Comment:
```suggestion
You will be able to add a Marketplace Provider in your Company and manage the accessibility of Marketplace resources connected to that Provider, directly from the Providers section in Console. All you need to do is to add a Marketplace Provider by clicking on the "Add Marketplace Provider connection" option – available in the Providers section – and then configure on the CMS all the Marketplace resources that you want to make available to other Companies. This can be done by specifying the providerId of the Marketplace Provider and editing the visibility property for each of those Marketplace resources.
```